Android/Shared/ITMAppConfig.gradle

```diff
@@ -30,7 +32,10 @@ static def getBoolProperty(properties, propertyName) {
 }
 
 task genITMAppConfig {
-    def itmAppConfig = new File("${rootDir}/app/src/main/assets/ITMApplication/ITMAppConfig.json")
+    def destDir = Paths.get("${rootDir}/app/src/main/assets/ITMApplication")
```

Comment:
genITMAppConfig depends on copyITMAssets which writes files to that directory. It would be better to have the folder being created in copyITMAssets or in both for resiliency.

Reply:
All I know is that the sync failed until I added this. I don't understand why, either.